kedro/pipeline/pipeline.py

```diff
@@ -371,33 +371,34 @@ def grouped_nodes(self) -> list[list[Node]]:
 
     @property
     def grouped_nodes_by_namespace(self) -> dict[str, dict[str, Any]]:
-        """Return a dictionary of the pipeline nodes grouped by top-level namespace with
-        information about the nodes, their type, and dependencies. The structure of the dictionary is:
```

Comment:
Any reason to update the docstring? I had it shorter in the beginning but made it longer based on PR feedback so might be good to leave it as it is?